### PR TITLE
AWS unified ingress

### PR DESCRIPTION
**Pull request by alexpirogovski**
Added unified ingress file for 5 services:
* mlrun-ui
* mlrun-api
* jupyter
* grafana
* nuclio
to use with AWS ALB (same as it was implemented in CFN stack)

### CODE REVIEW

File: charts/mlrun-ce/templates/_helpers.tpl [hunk 1]
```diff
@@ -226,3 +226,14 @@ Model monitoring DSN
...
+{{/*
+serviceAccounts for AWS
+*/}}
+{{- define "aws.serviceAccounts" -}}
+mlrun-api-aws-sa
+mlrun-jobs-aws-sa
+ml-pipeline-ui-sa
+ml-pipeline-sa
+argo-sa
+{{- end -}}
```
Comment (TomerShor) [inline charts/mlrun-ce/templates/_helpers.tpl:238]: where is this being used?

File: charts/mlrun-ce/values.yaml [hunk 2]
```diff
@@ -417,3 +417,9 @@ kube-prometheus-stack:
     fullnameOverride: state-metrics
   prometheus-node-exporter:
     fullnameOverride: node-exporter
+
+cloud:
+  aws:
+    enabled: false
+    roleArn: ~
```
Comment (TomerShor) [inline charts/mlrun-ce/values.yaml:424]: same here - where is it being used?

File: charts/mlrun-ce/templates/_helpers.tpl (hunk 1 above)
Comment (alexpirogovski) [reply charts/mlrun-ce/templates/_helpers.tpl:239]: Probably nowhere, just legacy from CFN era but I don't want to delete yet because no one knows for sure. Marked for deletion

File: charts/mlrun-ce/values.yaml (hunk 2 above)
Comment (alexpirogovski) [reply charts/mlrun-ce/values.yaml]: Correct, removed

File: charts/mlrun-ce/templates/jupyter-notebook/ingress.yaml [hunk 3]
```diff
@@ -17,7 +17,6 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  ingressClassName: nginx
```
Comment (TomerShor) [inline charts/mlrun-ce/templates/jupyter-notebook/ingress.yaml:20]: Why is this removed?
What is the default `ingressClassName` that will be used?

Comment (alexpirogovski) [reply charts/mlrun-ce/templates/jupyter-notebook/ingress.yaml]: Fixed

**Review by TomerShor — APPROVED** [review]
